test/regex/nullByteBug.chpl

```diff
@@ -4,14 +4,14 @@ var myRegex = new regex(b"a+");
 
 var str = b"oneatwo";
 
-writef("%t\n", str.replaceAndCount(myRegex, b"\x00"));
+writef("%ht\n", str.replaceAndCount(myRegex, b"\x00"));
 
-writef("%t\n", (new regex(b"\x00")):bytes);
-writef("%t\n", (new regex(b"a\x00")):bytes);
-writef("%t\n", (new regex(b"\x00a")):bytes);
+writef("%ht\n", (new regex(b"\x00")):bytes);
+writef("%ht\n", (new regex(b"a\x00")):bytes);
+writef("%ht\n", (new regex(b"\x00a")):bytes);
 
 var r3 = new regex(b"\x00+");
 
-writef("%t\n", r3:bytes);
+writef("%ht\n", r3:bytes);
 
-writef("%t\n", b"a\x00b\x00\x00c\x00\x00\x00".replace(r3, b"A"));
+writef("%ht\n", b"a\x00b\x00\x00c\x00\x00\x00".replace(r3, b"A"));
```

Comment:
What does the output look like with ``%t``?

Reply:
Ah, nevermind, I see that it would be something like ``my data`` for ``b"my data"``.

Reply:
Right, and the numbers are written in actual binary format.